tests/integration/json_test.go

```diff
@@ -30,6 +30,15 @@ var _ = Describe("odojsonoutput", func() {
 		helper.DeleteDir(tmpDir)
 	})
 
+	Context("odo machine create project json output", func() {
+		// odo project create newprojectjson -o json
+		It("should be able to create project and show output in json format", func() {
+			actual := helper.CmdShouldPass("odo project create newprojectjson -o json")
```

Comment:
Should this be `helper.CmdShouldPass("odo", "project", "create", "newprojectjson", "-o", "json")`?

Reply:
Yeah, you're right :) Updated!